scripts/run_upp

```diff
@@ -64,8 +64,8 @@ set -x
 # as recommended in the users guide where UPP will output.
 export TOP_DIR=/home/username
 export DOMAINPATH=${TOP_DIR}/test_case
-export UNIPOST_HOME=${TOP_DIR}/EMC_post
-export POSTEXEC=${UNIPOST_HOME}/bin
+export UNIPOST_HOME=${TOP_DIR}/UPP
```

Comment:
Could consider changing UNIPOST_HOME to UPP_HOME here and throughout.
 (L38, 67, 68, 198-199, 204-205, 208)